addons/marketing_card/tests/test_campaign.py

```diff
@@ -323,3 +323,56 @@ def test_campaign_ownership(self):
         with self.assertRaises(exceptions.AccessError):
             campaign_as_other.unlink()
         campaign_as_owner.unlink()
+
+    def test_mail_render_security(self):
+        """Asserts the security of the QWeb rendering of the field body_html of card.campaign
+
+        The security is based on the fact:
+        - `body_html` is a related to a `card.template.body`,
+        - `card.template` can only be edited by `base.group_system`,
+        - `body_html` is not `store=True, readonly=False`:
+          The user or manager cannot edit the body html on the campaign directly.
+
+        Hence, only admin can control the `body_html` of the campaign, hence it can be rendered without restriction.
+
+        If one of the above assumptions would change,
```

Comment:
Would move that above the _unrestricted_rendering attribute, makes more sense to have explanation in models than in tests (which can simply link the explanation done on model)